### PR TITLE
fix: \n rendered as line break in multi-line view

### PR DESCRIPTION
**Pull request by navyayy**
# Description
Now, the `\n` in the `message` will be rendered as line break.

# Testing
I have tested by ingesting logs with `\n` in the `message`.
image

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: static/js/event-handlers.js [hunk 1]
```diff
@@ -570,6 +570,25 @@ function logOptionMultiHandler() {
             
             colDef.cellStyle = {'white-space': 'normal'};
             colDef.autoHeight = true;
+            colDef.cellRenderer = function(params) {
+                console.log('params: ', params)
```
Comment (nkunal) [inline static/js/event-handlers.js:574]: remove log

**Review by nkunal — COMMENTED** [review]
@sonamgupta21 please review this

File: static/js/event-handlers.js (hunk 1 above)
Comment (navyayy) [reply static/js/event-handlers.js:574]: yeah sure

**Review by sonamgupta21 — COMMENTED** [review]
https://github.com/siglens/siglens/assets/71771131/6576f937-d0d0-4f4c-9e94-9ce2ce9a116d

It looks fine in wrap view, but in single line view, it does not allow horizontal scrolling.

**Comment by sonamgupta21** [comment]
I still see the issue
image

**Pull request closed without merging**